test/broadcasting.jl

```diff
@@ -220,7 +224,8 @@ end
     @test df[:, Not("x1")] == refdf[:, 2:end]
 
     dfv = @view df[1:2, 2:end]
-    @test_throws ArgumentError dfv[!, 1] .+= [0, 1] .+ 1
+    dfv[!, 1] .+= [0, 1] .+ 1
+    @test df.x2 == [5.5, 7.5, 6.5]
```

Comment:
Check the contents of the whole `df` while you're at it (here and elsewhere)? For example, the column could be re-added in the wrong place...

Reply:
done